SystemServices/CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ All notable changes to this RDK Service will be documented in this file.
 * Changes in CHANGELOG should be updated when commits are added to the main or release branches. There should be one CHANGELOG entry per JIRA Ticket. This is not enforced on sprint branches since there could be multiple changes for the same JIRA ticket during development. 
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
+## [3.1.2] - 2024-09-17
+### Added
```

Comment:
Pls update minor version and reset patch version. So move to "3.2.0". Details on versioning mentioned in the README - https://github.com/rdkcentral/RDKServices?tab=readme-ov-file#versioning

Reply:
Updated